x-pack/plugins/security_solution/public/detections/containers/detection_engine/alerts/api.ts

```diff
@@ -256,3 +259,19 @@ export const getHostMetadata = async ({
     resolvePathVariables(HOST_METADATA_GET_ROUTE, { id: agentId }),
     { method: 'GET', signal, version: '2023-10-31' }
   );
+
+/**
+ * Fetches suggested user profiles
+ */
+export const suggestUsers = async ({
```

Comment:
I think we defined this exact function somewhere else. Can we put it somewhere it can be reused?

Reply:
Removed! It was a leftover